app/models/container_group.rb

```diff
@@ -85,8 +85,6 @@ def disconnect_inv
     _log.info "Disconnecting Pod [#{name}] id [#{id}] from EMS [#{ext_management_system.name}]" \
     "id [#{ext_management_system.id}] "
     self.container_definitions.each(&:disconnect_inv)
-    self.old_ems_id = ems_id
-    self.ext_management_system = nil
```

Comment:
It make sense to unify the behavior and not delete the foreign keys. It will be a zombie-free land at some point :-)